landing-page/types/headnode/layouts/navbar.html.erb

```diff
@@ -84,5 +84,16 @@
               </li>
             <% end %>
         </ul>
+        <ul class="navbar-nav">
+            <li class="nav-item">
+              <div id="flight-account-menu"></div>
+            </li>
+            <li class="nav-item">
+              <div id="flight-account-menu-sm"></div>
+            </li>
+            <li class="nav-item">
+              <div id="flight-account-menu-modal-container"></div>
+            </li>
```

Comment:
These lines aren't needed.  They're used by the Flight Center version of flight-account-menu, but not by the websuite version.